recipes/yambo/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "yambo" %}
+{% set version = "5.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/yambo-code/yambo/archive/refs/tags/{{ version }}.tar.gz
+  sha256: a2cc0f880dd915b47efa0d5dd88cb94edffbebaff37a252183efb9e23dbd3fab
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - make
+    - m4
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+  host:
+    - openmpi
+    - fftw
+    - libblas
+    - liblapack
+    - wget
```

Comment:
Is wget needed in host? I would assume it’s more likely to be a build dependency? (Build if it’s used for downloads, host if you link against it)

if it is being used for downloads, they should probably be included in the sources section instead

Reply:
thanks, I think you are right - it will only be needed in `build`.

> if it is being used for downloads, they should probably be included in the sources section instead

I'm going to assume you meant "`build:`  section" here as well - please correct if you meant something else